open_url argument added to prevent browser from opening

aeims opened the pull request:
continuation of the first pull request: passing open_url=True will open browser and return True on success  else it will just return the button's URL to be processed via code.

aeims committed:
open_url argument added to prevent browser from opening

passing open_url=True will open browser and return True on success  else it will just return the button's URL to be processed via code.

Lonami commented:
Could you add the other PR's commit here? One PR should be enough for this change.

aeims committed:
adding open_url argument for Message.Button Class

adding necessary changes so it works with changes made in telethon.tl.Message.Button

open_url is a bool arg defualt to None (False) so clicking a url inline button will actually return the URL for further proccessing in code instead of returning True and opening it in browser.

(sorry for two pull requests I didn't notice patch numbers)

aeims commented:
It is done, sorry for inconvenience

aeims committed:
adding docstring for open_url seems needed

other args had descriptions so I felt the urge to write some too.

Lonami commented:
Technically a breaking change to change the default, but frankly I doubt anyone was relying on the old behaviour.
